src/modals/import-account-modal/import-account-type/import-account-type.tsx

```diff
@@ -0,0 +1,56 @@
+import { Formik } from 'formik';
+import React, { FC } from 'react';
+import { View } from 'react-native';
+
+import { ButtonLargePrimary } from '../../../components/button/button-large/button-large-primary/button-large-primary';
+import { ButtonLargeSecondary } from '../../../components/button/button-large/button-large-secondary/button-large-secondary';
+import { ButtonsContainer } from '../../../components/button/buttons-container/buttons-container';
+import { Divider } from '../../../components/divider/divider';
+import { InsetSubstitute } from '../../../components/inset-substitute/inset-substitute';
+import { Label } from '../../../components/label/label';
+import { ScreenContainer } from '../../../components/screen-container/screen-container';
+import { EventFn } from '../../../config/general';
+import { FormRadioButtonsGroup } from '../../../form/form-radio-buttons-group';
+import { ImportAccountTypeEnum, ImportAccountTypeValues } from '../../../interfaces/import-account-type';
+import { useNavigation } from '../../../navigator/hooks/use-navigation.hook';
+import { formatSize } from '../../../styles/format-size';
+import { importAccountTypeInitialValues, importAccountTypeValidationSchema } from './import-account-type.form';
+
+interface Props {
+  onSubmit: EventFn<ImportAccountTypeValues>;
+}
+
```

Comment:
here for ex.
https://github.com/madfish-solutions/templewallet-mobile/pull/172/files#r684012255